modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -685,6 +685,64 @@ definition {
 		ObjectAdmin.assertObjectActionsNotPresent(actionName = "Action Name 2");
 	}
 
+	@description = "LPS-156566 - Verify an Action with Groovy Script can be triggered after deleting an entry."
+	@priority = "4"
+	test CanTriggerActionDeleteWithGroovyScript {
+		task ("Given an object witha a field is created") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 156",
+				objectName = "CustomObject156",
+				pluralLabelName = "Custom Objects 156");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Integer",
+				fieldLabelName = "Custom Integer",
+				fieldName = "customInteger",
+				fieldType = "Integer",
+				isRequired = "false",
+				objectName = "CustomObject156");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject156");
+		}
+
+		task ("When an action is created using Groovy Script and is triggered when an object field is deleted") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 156");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionName = "Action Name",
+				active = "true",
+				scriptValidation = "println 'Success'",
```

Comment:
I think this groovy script is good, but how do you validate this `println` in the functional test?